metrics/network/network-metrics-iperf3.sh

```diff
@@ -0,0 +1,540 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test measures the following network essentials:
+# - bandwith simplex
+# - bandwith duplex
+# - jitter
+#
+# These metrics/results will be got from the interconnection between
+# a client and a server using iperf tool.
+# The following cases are covered:
+#
+# case 1:
+#  container-server <----> container-client
+#
+# case 2"
+#  container-server <----> host-client
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/lib/network-common.bash"
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# Port number where the server will run
+port="${port:-5201}"
+fwd_port="${fwd_port:-$port:$port}"
+# Image name
+image="${image:-gabyct/network}"
+# Measurement time (seconds)
+transmit_timeout="${transmit_timeout:-5}"
```

Comment:
For anything that is configurable or might be good to know/reference/document in the future (such as maybe also the iperf3 versions being used) - can we inject those into a separate JSON section in the results files please. I particular this timeout sprang to mind - but there may be other things like bandwidth and packet size settings.
You can see what I mean with the `save_config` func over in the docker density test for instance: https://github.com/kata-containers/tests/blob/master/metrics/density/docker_memory_usage.sh#L290